modules/pom.xml

```diff
@@ -1,120 +1,132 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please check your IDE config, looks like there are some formatter issues...

Reply:
Oh is it ? i didn't know about that .

Reply:
Yes I have a lot of white space changes in XMLs you edited.